### PR TITLE
fix: missing fmt in the go program and improve the "Pulumi Fundamentals" tutorial

### PR DESCRIPTION
**Pull request by scraly**
## Description

I fixed a missing "fmt" import in the go program, without i the tutorial is not working.
I have also improved the tutorial with more information to guide the user.

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] I have manually confirmed that all new links work.

### CODE REVIEW

File: themes/default/content/learn/pulumi-fundamentals/create-docker-images/index.md [hunk 1]
```diff
@@ -434,6 +436,8 @@ mongo_image = docker.RemoteImage("mongo_image",
 
 {{% choosable language go %}}
 
+Add this code at the bottom of your program, just before the last return nil statement:
```
Comment (scottslowe) [inline themes/default/content/learn/pulumi-fundamentals/create-docker-images/index.md:439]:
```suggestion
Add this code at the bottom of your program, just before the last `return nil` statement:
```

**Review by scottslowe — CHANGES_REQUESTED** [review]
One very minor suggested change; otherwise, looks good and thank you very much for your contribution!

**Comment by scraly** [comment]
Updated 🙂

**Review by scottslowe — APPROVED** [review]
Approved, thanks again for the contribution!

**Pull request merged**